tgui/packages/tgui/interfaces/AlertModal.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file
+ * @copyright 2020 bobbahbrown (https://github.com/bobbahbrown)
+ * @license MIT
+ */
+
+import { clamp01 } from 'common/math';
+import { useBackend } from '../backend';
+import { Box, Button, Flex } from '../components';
+import { Window } from '../layouts';
+
+export const AlertModal = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    title,
+    message,
+    buttons,
+    timeout,
+  } = data;
+
+  return (
+    <Window
+      title={title}
+      width={350}
+      height={150}
+      resizable>
+      {timeout !== undefined && <Loader value={timeout} />}
+      <Window.Content>
+        <Flex direction="column" height="100%">
+          <Flex.Item grow={1}>
+            <Flex
+              direction="column"
+              className="AlertModal__Message"
+              height="100%">
+              <Flex.Item>
+                <Box m={1}>
+                  {message}
+                </Box>
+              </Flex.Item>
+            </Flex>
+          </Flex.Item>
+          <Flex.Item my={2}>
+            <Flex className="AlertModal__Buttons">
+              {buttons.map(button => (
+                <Flex.Item key={button} mx={1}>
+                  <Button
+                    px={3}
```

Comment:
Sanity check on need for this. Buttons should have decent padding for the case of simple text already.

Reply:
px=3 better, this isn't for spacing, this is for button size.
removed (comp to OP):
![image](https://user-images.githubusercontent.com/4741640/104782568-43f7d580-5739-11eb-953c-de2bb174f74f.png)

Reply:
I gotcha, go ahead and keep the `px={3}` here, I'll take a think about different "levels" of Button at some point with regards to standardizing. Will probably implement a wrapper component that does a bunch of the "presets" like this if it comes up again, no need to refactor for first use-case.